src/components/AboutModal.tsx

```diff
@@ -58,18 +57,21 @@ const AboutModal: React.FC<AboutModalProps> = ({ isOpen, onClose }) => {
       brandImageAlt="Assisted Installer Logo"
     >
       <TextContent>
-        <TextList component="dl">
-          <TextListItem component="dt">Assisted Installer UI version</TextListItem>
-          <TextListItem component="dd">
-            {VERSION} {GIT_SHA ? `(${GIT_SHA})` : ''}
-          </TextListItem>
-          {Object.keys(versions).map((version) => (
-            <>
-              <TextListItem component="dt">{SERVICE_LABELS[version] || version}</TextListItem>
-              <TextListItem component="dd">{versions[version]}</TextListItem>
-            </>
-          ))}
-        </TextList>
+        <DetailList>
+          <>
```

Comment:
Is this needed?

Reply:
yes because DetailList expect children to be either ReactChild or ReactChild[]
`children: ReactChild | (ReactChild | undefined)[];`

Reply:
The resulting type would be `{ children: (Element | Element[])[]; }`. I agree that this is the easiest workaround. Or wrapping just the generated array within `<>`